hail/python/hail/expr/functions.py

```diff
@@ -7070,6 +7070,106 @@ def coerce_endpoint(point):
     )
 
 
+@typecheck(path=builtins.str, point_or_interval=expr_any, entries_name=builtins.str)
+def query_matrix_table(path, point_or_interval, entries_name='entries_array'):
```

Comment:
How would you feel about `query_matrix_table_rows`? Might we have need sometime for something analogous that queries the columns?

Reply:
Sure. I'll rename. After some thought, I feel that name aligns best with our other API naming conventions. Though if we were to be truly accurate, it'd be something like `query_matrix_table_localized_rows` but that's just unwieldy.